CMakeLists.txt

```diff
@@ -312,11 +321,18 @@ if(TRITON_ONNXRUNTIME_DOCKER_BUILD)
     set(_GEN_FLAGS ${_GEN_FLAGS} "--ort-openvino=${TRITON_BUILD_ONNXRUNTIME_OPENVINO_VERSION}")
   endif() # TRITON_ENABLE_ONNXRUNTIME_OPENVINO
 
+  set(ENABLE_GPU_EXTRA_ARGS "")
+  if(${TRITON_ENABLE_GPU})
+    set(ENABLE_GPU_EXTRA_ARGS "--enable-gpu")
+    message(STATUS "Using GPU")
+  endif() # TRITON_ENABLE_GPU
+
+  message(STATUS ${FETCHCONTENT_BASE_DIR}/repo-common-src)
```

Comment:
What's the value of adding these messages? I think these can be removed or made verbose?

Reply:
Sorry I was trying something. This has been deleted